File: readability/readability.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>
#include <ctype.h>


int main(void)
{
    string text = get_string ("Text: ");
    printf("Text: %s\n", text);

    int letters, words, sentances;

    for (int i = 0; text[i] != 0; i++){

        if (text[i] == 32)
        {
            words++;
        }
        else if ((text[i] == 46) || (text[i] == 33) || (text[i] == 63))
        {
            sentances++;
        }
        else if (isalpha(text[i]))
        {
            letters++;
        }
    }

    words ++;

    float L = (float) letters / (100.0 * words);
    float S = (float) sentances / (100.0 * words);

    int index = 0.0588 * L - 0.296 * S - 15.8;

    printf("%i\n", index);

}

File: sentimental-readability/readability.py
import cs50

text = cs50.get_string("Text: ")
L = 0
S = 0
letters = 0
sentences = 0
words = 1

for i in text:
    if text[i].isalplha:
        letters += 1
    elif (text[i] == ".") or (text[i] == ".") or (text[i] == "?"):
        sentences += 1
    elif text[i] == " ":
        words += 1

l = letters / (100 * words)
S = sentences /


index = 0.0588 * Letters - 0.296 * S(Sentances/100 words) - 15.8
(words are spaces)

print("Grade {}".format(index))
